src/pages/index.tsx

```diff
@@ -1,4 +1,6 @@
 import GoogleMapReact from "google-map-react";
+import React from "react";
+import Marker from "../../components/Marker/Marker";
 
 export default function Page() {
   return (
```

Comment:
Let's move:

```tsx
<GoogleMapReact
        bootstrapURLKeys={{ key: process.env.NEXT_PUBLIC_GOOGLE_MAPS_KEY }}
        defaultCenter={{ lat: 30.2672, lng: -97.7393 }}
        defaultZoom={10}
      />
      <Marker
        lat={30.2672}
        lng={-97.7393}
        name="Crag Climbing 🧗🏾‍♂️"
        color="blue"
      />
```

Into the `map.tsx` component, and then require it in this file.

Reply:
I went ahead and pushed the change, so I can't find the marker still hahah but it is compiling.  We can look at it today before you accept the merge